components/markdown_image.test.jsx

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+
+import configureStore from 'redux-mock-store';
+import {shallow} from 'enzyme';
+
+import MarkdownImage from 'components/markdown_image.jsx';
+
+describe('components/MarkdownImage', () => {
+    const mockStore = configureStore();
+    const initialState = {
+        entities: {
+            general: {
+                config: {
+                    DefaultClientLocale: 'en',
+                },
+            },
+        },
+    };
+    const store = mockStore(initialState);
+    it('should match snapsnot for SizeAwareImage dimensions', () => {
+        const imageMetadata = {format: 'jpg', frame_count: 0, width: 100, height: 100};
+        const wrapper = shallow(
+            <MarkdownImage
+                imageMetadata={imageMetadata}
+                src='path/image'
+            />
+        );
+
+        const childrenWrapper = wrapper.dive({context: {store}}).props().children('safeSrc');
```

Comment:
You don't actually need to dive here, so we can get rid of all the store setup
```suggestion
        const childrenWrapper = wrapper.props().children('safeSrc');
```

Reply:
Done

Reply:
Thanks for the clarification @hmhealey, my code snippet was from an earlier iteration where i had the markdown_image connected to the store - no longer the case though -, and the dive was needed. Sorry @sudheerDev that i missed to note this when i commented earlier.